### PR TITLE
home task 1

### PR DESCRIPTION
**Pull request by mkozhan**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -1 +1,84 @@
...
+    });
+    return(    
+        <form onSubmit = {event=>
+                {
+                    event.preventDefault();
+                    onSubmitFunc({...newUser, id:Date.now()});
+                }
+            }
+        >
+            <input placeholder='User Name' onChange={event => newUser.userName = event.target.value }/>
```
Comment (pannkotsky) [inline app.js]: values in state should be only updated via callback obtained from useState, not directly. like this:
```suggestion
            <input placeholder='User Name' value={newUser.userName} onChange={event => setNewValues({...newUser, userName: event.target.value}) }/>
```

File: app.js [hunk 2]
```diff
@@ -1 +1,84 @@
...
+                        usersData.map(user => <UserRender key={user.id} userData = {user} />)
+                    }
+               </tbody>
+        </table>
+    </div>
+);
+
+const FormForUSer =({onSubmitFunc})=>
+{
+    const[newUser,setNewValues] = React.useState(
```
Comment (pannkotsky) [inline app.js]: the convention is to name this pair of variables like: `x, setX`, e.g. `newUser, setNewUser` or `newValues, setNewValues`

File: app.js [hunk 3]
```diff
@@ -1 +1,84 @@
...
+            <input placeholder='User Name' onChange={event => newUser.userName = event.target.value }/>
+            <input placeholder='User Age Name' type = "number" onChange={event => newUser.userAge = event.target.value }/>
+            <input placeholder='User Phone' onChange={event => newUser.userPhone = event.target.value }/>
+            <button type="submit">Send new Values</button>
+        </form>
+    );    
+}
+
+const App = () => {
+    const [users,invokeReload] = React.useState(usersList);
```
Comment (pannkotsky) [inline app.js]: see the note about naming

File: app.js (hunk 1 above)
Comment (mkozhan) [reply app.js]: Done

File: app.js (hunk 2 above)
Comment (mkozhan) [reply app.js]: Done

File: app.js (hunk 3 above)
Comment (mkozhan) [reply app.js]: Done

**Pull request merged**